app/src/ui/multi-commit-operation/base-rebase.tsx

```diff
@@ -0,0 +1,83 @@
+import { isRebaseConflictState, RebaseConflictState } from '../../lib/app-state'
+import {
+  instanceOfIBaseRebaseDetails,
+  MultiCommitOperationKind,
+} from '../../models/multi-commit-operation'
+import { BaseMultiCommitOperation } from './base-multi-commit-operation'
+
+export abstract class BaseRebase extends BaseMultiCommitOperation {
+  protected abstract conflictDialogOperationPrefix: string
+  protected abstract rebaseKind: MultiCommitOperationKind
+
+  protected onContinueAfterConflicts = async (): Promise<void> => {
+    const {
+      repository,
+      dispatcher,
+      workingDirectory,
+      state,
+      conflictState,
+    } = this.props
+    const { operationDetail, originalBranchTip } = state
+
+    const targetBranchName =
+      conflictState !== null && isRebaseConflictState(conflictState)
+        ? conflictState.targetBranch
+        : null
+    const baseBranchName =
+      conflictState !== null && isRebaseConflictState(conflictState)
+        ? conflictState.baseBranch || ''
+        : ''
```

Comment:
Just a note: I think this doesn't work yet ([but works in a newer version of TypeScript](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4-beta/#cfa-aliased-conditions)), at some point we can extract part of the condition to a constant and TS will infer the types as expected :_)
```suggestion
    const isRebaseConflictState = conflictState !== null && isRebaseConflictState(conflictState)
    const targetBranchName =
      isRebaseConflictState
        ? conflictState.targetBranch
        : null
    const baseBranchName =
      isRebaseConflictState
        ? conflictState.baseBranch || ''
        : ''
```